io.github.aginies.watermark.yml

```diff
@@ -0,0 +1,24 @@
+app-id: io.github.aginies.watermark
+runtime: 'org.freedesktop.Platform'
+runtime-version: '24.08'
+sdk: 'org.freedesktop.Sdk'
+command: watermark_app
+finish-args:
+  - --share=ipc
+  - --socket=x11
```

Comment:
if it is gtk3 this should be fallback-x11+wayland

Reply:
done in latest PR. default wayland, fallback to x11.